Agents/Xamarin.Interactive/Representations/RepresentationProvider.cs

```diff
@@ -38,7 +39,25 @@ public virtual IEnumerable<object> ProvideRepresentations (object obj)
             return false;
         }
 
-        protected virtual bool TryFindMatchingRepresentation<TResolvedType, TRepresentationType> (
+        protected virtual bool TryConvertRepresentation<TResolvedType, TRepresentationType> (
```

Comment:
I wonder if these generic parameters shouldn't be the other way around, like they are in the `converter` func.